lib/annotate_rb/runner.rb

```diff
@@ -23,23 +23,18 @@ def running?
     end
 
     def run(args)
-      config_file_options = ConfigLoader.load_config
       parser = Parser.new(args, {})
-
       parsed_options = parser.parse
       remaining_args = parser.remaining_args
 
-      options = config_file_options.merge(parsed_options)
+      AnnotateRb::RakeBootstrapper.call
```

Comment:
It's been a while since I've touched this code, but what's the thinking around the reordering?

Reply:
I needed to re-order to use `Rails.root`. But since I found out this gem is used for non-rails projects, I've restored the code.